kernel/component-manager/src/main/java/org/sakaiproject/util/serializable/SakaiCglibSubclassingInstantiationStrategy.java

```diff
@@ -0,0 +1,228 @@
+/**********************************************************************************


```

Comment:
As noted, nearly a copy of the Spring CglibSubclassingInstantiationStrategy.java, except that we use a Serialized form of the NoOp